### PR TITLE
adding all get-al files in single directory

### PR DESCRIPTION
**Pull request by rushikesh-kulkarni**

### CODE REVIEW

File: space-cli/modules/all.go [hunk 1]
```diff
@@ -163,6 +170,17 @@ func GetAllProjects(c *cli.Context) error {
 }
 
 func createConfigFile(pos, commandName string, objs []*model.SpecObject) error {
+
+	ex, err := os.Executable()
+	if err != nil {
+		panic(err)
```
Comment (sharadregoti) [inline space-cli/modules/all.go]: return error here

**Comment by YourTechBud** [comment]
Fixes bugs with #738 
Fixes #737

**Comment by YourTechBud** [comment]
@sharadregoti make sure we fix `space-cli apply` directory in the PR itself

**Comment by sharadregoti** [comment]
ok

**Comment by YourTechBud** [comment]
@sharadregoti this entire PR seems to be incorrect. I suggest we scrap this and let someone else do it

**Pull request closed without merging**